ipn/ipnlocal/local.go

```diff
@@ -6453,8 +6453,16 @@ func suggestExitNode(report *netcheck.Report, netMap *netmap.NetworkMap, r *rand
 	if report.PreferredDERP == 0 {
 		return res, ErrNoPreferredDERP
 	}
+	var allowedCandidates set.Set[string]
+	if allowed, ok := syspolicy.GetStringArray(syspolicy.AllowedSuggestedExitNodes, []string{}); ok == nil {
+		allowedCandidates = set.SetOf(allowed)
+	}
 	candidates := make([]tailcfg.NodeView, 0, len(netMap.Peers))
+	isAuto := netMap.SelfNode.CapMap().Has(tailcfg.NodeAttrAutoExitNode)
```

Comment:
Maybe call this `canAuto` -- from what I understand, that signals that the auto option is possible, not that it's in use.